scss/bootstrap/bootstrap.scss

```diff
@@ -1,19 +1,17 @@
-@import "vendor/text-angular/text-angular";
-@import "vendor/angular-xeditable/angular-xeditable";
-@import "vendor/font-awesome/font-awesome";
-@import "vendor/perfect-scrollbar/main";
-@import "vendor/ui-bootstrap/ui-bootstrap";
-@import "vendor/ui-select/ui-select";
-@import "vendor/select2/select2-bootstrap";
+@import '../angular/vendor/*';
+@import 'vendor/font-awesome/font-awesome';
+@import 'vendor/perfect-scrollbar/main';
+@import 'vendor/select2/select2-bootstrap';
 
-@import "overrides/variables";
-@import "fonts/*";
-@import "mixins/*";
+@import 'overrides/variables';
+@import 'fonts/*';
+@import 'mixins/*';
 
-@import "SCSSROOT/org.civicrm.shoreditch/base/scss/bootstrap";
-@import "overrides/style/*";
-@import "overrides/vendors/*";
-@import "overrides/crm/*";
-@import "variants/*";
-@import "components/*";
-@import "helper";
+@import 'SCSSROOT/org.civicrm.shoreditch/base/scss/bootstrap';
+@import '../angular/overrides/*';
+@import 'overrides/style/*';
+@import 'overrides/vendors/*';
+@import 'overrides/crm/*';
+@import 'variants/*';
+@import 'components/*';
+@import 'helper';
```

Comment:
this is something i noticed the first time but forgot to point out: this partial can't be here. There shouldn't be any scss partial in the root other than this one. Please find a better location for the file (or even better, find an already existing partial for the class(es) that `helper` currently contains

Reply:
Based on our discussion on hipchat, moving the helper file to `utilities` folder.